crates/api/src/chronicle_graphql/mod.rs

```diff
@@ -806,7 +862,7 @@ where
             let app = match sec.jwks_uri {
                 Some(jwks_uri) => {
                     const CACHE_EXPIRY_SECONDS: u32 = 100;
-                    tracing::debug!("API endpoint authentication uses {}", jwks_uri);
+                    tracing::debug!("API endpoint authentication uses {:?}", jwks_uri);
```

Comment:
```suggestion
                    tracing::debug!("API endpoint authentication uses {jwks_uri:?}");
```
while we're here anyway.